lib/charms/operator_libs_linux/v2/snap.py

```diff
@@ -970,7 +993,7 @@ def ensure(
         channel = "latest"
 
     if state in ("present", "latest") or revision:
-        return add(snap_names, SnapState(state), channel, classic, cohort, revision)
+        return add(snap_names, SnapState(state), channel, classic, devmode, cohort, revision)
```

Comment:
Same comment here about keyword arguments. A call site with 7 positional arguments, yucky. :-) Again, not your fault -- I realize you're just adding to it.

Reply:
Done